lib/wisper/version.rb

```diff
@@ -1,3 +1,3 @@
 module Wisper
-  VERSION = "2.0.1"
+  VERSION = "3.0.0"
```

Comment:
could you please revert this change? it's better to leave it to maintainers to decide when gem's version has to be updated

Reply:
Agreed, I can cut a release in a separate commit, just make sure to mark it as backwards incompatible in the CHANGELOG. Thanks!